octobot_trading/personal_data/orders/channel/orders_updater.py

```diff
@@ -113,13 +113,15 @@ async def _closed_orders_fetch_and_push(self, limit=ORDERS_UPDATE_LIMIT) -> None
     async def update_order_from_exchange(self, order,
                                          should_notify=False,
                                          wait_for_refresh=False,
-                                         force_job_execution=False):
+                                         force_job_execution=False,
+                                         create_order_producer_if_missing=True):
```

Comment:
Param is not used ?

Reply:
This is not matching OrdersProducer.update_order_from_exchange params without  `create_order_producer_if_missing`.